### DISCUSSION TITLE
Does triton support pulling model repo from OSS

### DISCUSSION THREAD

**Discussion by wyoung1** [triton-inference-server/server#5767] [Q&A]
Does triton support pulling model repo from OSS just like this:

tritonserver --model-repository=oss://

I use this command but returns error:

error: creating server: Internal - failed to stat file oss://

If it supports OSS, what environment values should I set?

**Comment by tanmayv25** [comment]
The supported locations for the model repo are all documented here:
https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_repository.md#model-repository-locations

**Comment by tankroom** [comment]
Hello, may I ask if this issue has been resolved? How was it resolved? I also encountered the same problem
